tests/unit/plugins/module_utils/test_attachment.py

```diff
@@ -179,44 +179,50 @@ def test_duplicate(self, tmp_path):
 
 class TestAttachmentAreChanged:
     def test_unchanged(self):
-        assert attachment.are_changed(
-            [
-                {"hash": "hash", "file_name": "attachment_name.txt"},
-                {"hash": "hash", "file_name": "another_file_name.txt"},
-            ],
-            {
-                "attachment_name.txt": {
-                    "path": "some/path/file_name.txt",
-                    "type": "text/markdown",
-                    "hash": "hash",
-                },
-                "another_file_name.txt": {
-                    "path": "some/path/another_file_name.txt",
-                    "type": "text/plain",
-                    "hash": "hash",
+        assert (
```

Comment:
Did you run a `make format`? This file needs to be checked out from main after a `make format`, otherwise the lint will crash next time.

Reply:
It failed on black locally, so I ran my local black against it. It seems to pass `make format` now, and checking the file now from `main` subsequent `make format` changes it again.

I have noticed that black has subtle differences between versions.

Reply:
I'm running with black-24.1.1-1.fc40.noarch

Reply:
the thing is the lint is checking out `main` instead of this branch, so the green in lint workflow is misleading.  You are running lint against the old version :).
I've always checkedout this file after a `make format`. 
And yes, you need the exact version of black.

Reply:
reverted